src/figuresClasses.ts

```diff
@@ -1,11 +1,106 @@
-export interface Figure {}
+/* eslint-disable @typescript-eslint/lines-between-class-members */
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
+  private sideA: number;
+  private sideB: number;
+  private sideC: number;
+
+  constructor(
+    public color: Color,
+    sideA: number,
+    sideB: number,
+    sideC: number,
```

Comment:
```suggestion
  constructor(
    public color: Color,
    private sideA: number,
    private sideB: number,
    private sideC: number,
```
You can do it inside constructor